utilities_general.py

```
@@ -1,3 +1,1903 @@
+def get_hashes_from_db():
+    check_previous_search =  "SELECT id_pdf, hash, page, x0, y0, x1, y1, original_path, saved_as, parent_alias FROM Anexo_Eletronico_Pdf_Hashes"
+    sqliteconn = None
+    global_settings.splash_window.label['text'] = "Carregando relação Links->Hashes..."
+    global_settings.splash_window.label.update_idletasks()
+    try:
+        sqliteconn = utilities_general.connectDB(str(global_settings.pathdb))
+        cursor = sqliteconn.cursor()
+        cursor.custom_execute(check_previous_search)    
+        records = cursor.fetchall()
+        
+        for idpdf, hash, pagenum, x0, y0, x1, y1 in records:
```

Ruff (0.15.10)

[error] Variable `hash` is shadowing a Python builtin

(A001)